types/prismic-dom/prismic-dom-tests.ts

```diff
@@ -0,0 +1,3 @@
+import prismicDom = require("prismic-dom");
```

Comment:
In the documentation Prismic-dom appears to have a default export but this dts doesn't reflect that.

import PrismicDOM from "prismic-doc" failes as no default is defined.